src/posts/2021-09-08-making-links-accessible.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Making links accessible
+description: Tips on how to make links accessible.
+category: How to
+author: Pascal Akunne
+date: 2021-09-08
+further_reading:
+  - title: Links and Hypertext
+    url: https://webaim.org/techniques/hypertext/
+    source:  WAIM Web Accessibility In Mind
+    year: 2021
+  - title: Link Purpose (Link only)
+    url: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html
+    source:  WCAG 2.0
+    year: 2016
+  - title: Links
+    url: https://accessibility.umn.edu/what-you-can-do/start-7-core-skills/links
+    source: University of Minnesota
+    year: 2021
+tags:
+  - howto
+---
+
+Links are one of the primary elements in web development. Links can be made accessible by making them clear, descriptive and purposeful out of context. This helps break accessibility barriers for both people with visual disability and clear sighted, as well as people with cognitive disability. Links should be made obvious so people can understand what is means. By default, links are underlined by the browser. 
+
+
+## Guidelines for making links accessible
+
+### Don'ts
+
+- Careless use of ambiguous link texts like `Click here`, `More...`, `Link` are not appropriate and should be avoided. Rather, a text that describes the purpose of that link should be used. 
+- Do not use a URL as your link text. This makes it difficult for most screen readers to understand, they read each letter of the URL. 
+- Link texts are best not repeated to avoid disorientation for users with cognitive disability. 
+- Links should not be made to look like a button, to avoid confusion. Links navigate to webpages while buttons perform other actions like submitting forms. 
+- The `href` attribute should not be empty. Some developers use the pound sign (`#`) as the value for the `href` attribute. There's no point in providing a link without a destination. 
+
+### Dos
+
+- The `aria-label` attribute should be used. Because it provides a descriptive link text for people that are visually impaired.
+```
+<a href="https://w3schools.com/html" aria-label="Read more on HTML" >Read more</a>
+```
+- For easy readability, link texts should be underlined. Although, traditionally, link texts are underlined with a color by the browser.
+- Use the `cursor: pointer` declaration on your CSS. This helps to identify a link on a webpage using the `a:hover` style. 
+- Links should be made easy to navigate to and select using the `tab` key and `enter` key respectively. 
```

Comment:
again this is default behavior, so maybe revise this wording to indicate that... as there shouldn't be anything special a developer would need to do here.